css-cascade/css-cascade/css-cascade-quiz-notes.md

```diff
@@ -0,0 +1,57 @@
+# css-cascade-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What are the four components of "the Cascade".
+  The four components of "the Cascase" are:
+
+  1. selectors
+  2. properties
+  3. values
+  4. specificity
```

Comment:
Not quite! It's referring to the components of the "Cascade". Source order, inheritance, etc..